scripts/globals/mobskills/head_snatch.lua

```diff
@@ -1,38 +1,32 @@
 ---------------------------------------------
 --  Head Snatch
---
 --  Description: Grabs a single target's head.
 --  Type: Physical
 --  Utsusemi/Blink absorb: Ignores shadows
 --  Range: Melee
 --  Notes: Only used by Gurfurlur the Menacing. Reduces HP to 10%.
 ---------------------------------------------
 
-require("scripts/globals/settings");
 require("scripts/globals/status");
 require("scripts/globals/monstertpmoves");
 
 ---------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-  if(mob:getFamily() == 316) then
-    local mobSkin = mob:getModelId();
-
-    if (mobSkin == 1867) then
-        return 0;
-    else
-        return 1;
-    end
-  end
     return 0;
 end;
 
 function onMobWeaponSkill(target, mob, skill)
```

Comment:
Another one of those **not sure if correct** moments, but there is some anecdotal evidence that DMG reduction from PLD sentinel/gear can change the amount stabbed off the player:

- http://ffxiclopedia.wikia.com/wiki/Gurfurlur_the_Menacing
- https://www.bluegartr.com/threads/66344-Gurfurlur-the-Menacing

May want to get it verified .